### DISCUSSION TITLE
Scrolling "Tabs" in top window, when many tabs are open and available space is not sufficiant to show them all

### DISCUSSION THREAD

**Discussion by BrommD** [1Remote/1Remote#378] [Ideas]
Hi

I have a lot of sessions open simultaneously and grouped them into one desktop window of 1Remote. The issue is, when too many tabs are arranged on the tab of such window, the "older" ones or that ones at the edge of the tab bar simply are visually pushed away to the "left/right" (NIL). I cannot access them anymore.

This also happens, when I resize the window to smaller size, Tabs are keeping its size and hence most edged  ones are hidden left/right away.
i.e.
![image](https://user-images.githubusercontent.com/33422643/225035261-954853dd-14bc-4280-b8ed-867b57e8309b.png)
but
![image](https://user-images.githubusercontent.com/33422643/225035458-05f075ca-7674-4d0e-94cd-249d95c3370a.png)

Any chance to "scroll the tab bar" without resizing the main window to most large option?

A workaround is to grab one and move left/right over tab bar to make the application think, I want to reorder the tab
![image](https://user-images.githubusercontent.com/33422643/225037237-eccb5efb-a1bb-4471-80bd-31c3b6f2ca6a.png)

but this is not really a solution.

thanks

Dejan

**Comment by majkinetor** [comment]
1. One option would be to allow mouse scroll weel above the tabs.
2. Another one would be to allow tabs to be shown as left/right panel, with regular scroll. This could be 1RM option (tab bar placement). This is better for people who have lots of tabs, as its more natural to scroll veritical then horizontal list. 
3. Third option would be to always show all tabs by cutting their width to required size (browsers do this, I do not like it personally).
4. Search for open tabs could come handy in any option.

**Reply by BrommD** [reply]
I second the second option :)

**Comment by VShawn** [comment]
Move your mouse on tab header, you can scroll the bar by mouse wheel

And there a scroll bar here when mouse hover over.

![image](https://user-images.githubusercontent.com/10143738/225175686-18f521d9-cf9e-4042-a375-a99f537dee44.png)

**Comment by BrommD** [comment]
U're right. Never "saw" this before, sry - me blind! So, for now, this is quite good to work. Any extension would be nice, but this is OK to work.
Thanks for "head-up"

Dejan

**Comment by majkinetor** [comment]
Didn't know that too :)
